tests/ast/conftest.py

```diff
@@ -29,6 +33,13 @@ def pytest_addoption(parser):
         default=False,
         help="If set, overwrite test files with the actual output as the expected output.",
     )
+    parser.addoption(
+        "--encoding",
+        action="store",
+        default=default_encoding(),
+        type=str,
+        help="Encoding for AST expectation output, values can be 'json' (default) or 'b64' for base64.",
```

Comment:
I think we actually prefer textproto over JSON. Also, I'm not really sure we have a good reason to keep base64 around. We're making this change because it's fragile, to begin with.

Reply:
Updated to textproto